dives/DL7.zxu

```diff
@@ -1,10 +1,14 @@
 FSH|^~\&{}|ANST01^12X456^A|ZXU|20180106163705+02:00|
 ZRH|^~\&{}|||MFWG|ThM|C|bar|L|
-ZDH|1|1|I|V|20180102101000|27|11|FO2|||
+ZDH|1|1|I|QS|20180101101000|27|11|FO2|||
+ZDT|1|1|10.0|20180101102000|25||
+ZDH|2|2|I|V|20180102101000|27|11|FO2|||
 ZDP{
 |0|1|1||||
 |60|10|||||
 |3300|10|||||
 |3600|0|||||
 ZDP}
-ZDT|1|1|10.0|20180102110000|25||
+ZDT|2|2|10.0|20180102110000|25||
+ZDH|3|3|I|QS|20180103101000|28|11|FO2|||
+ZDT|3|3|10.0|20180103102000|26||
```

Comment:
could be combined with commit `DL7/zxu sample dive` ?

Reply:
I implemented this in steps keeping the sample dive and actual functionality in sync. But of course, this could be squashed afterwards.